src/lib/ecore/ecore_time.c

```diff
@@ -55,14 +55,30 @@ ecore_time_get(void)
 EAPI double
 ecore_time_unix_get(void)
 {
-//#ifdef HAVE_GETTIMEOFDAY
+
+
+   //struct timeval {
+     //          time_t tv_sec; / * segundos * /
+       //        suseconds_t tv_usec; / * microssegundos * /
+         //  };
+
+#ifdef HAVE_GETTIMEOFDAY
+
    struct timeval timev;
 
    gettimeofday(&timev, NULL);
    return (double)timev.tv_sec + (((double)timev.tv_usec) / 1000000);
-//#else
-//# error "Your platform isn't supported yet"
-//#endif
+   
+#else
+  
+   SYSTEMTIME lpSystemTime;
+   
+   // for Local time, replace GetSystemTime with GetLocalTime 
+   //GetLocalTime(&lpSystemTime);
+   GetSystemTime(&lpSystemTime);
```

Comment:
It seems that `GetSystemTime` by itself isn't good compliance with its POSIX version, which returns time since [epoch](https://man7.org/linux/man-pages/man2/gettimeofday.2.html) ([1970-01-01 00:00:00](https://man7.org/linux/man-pages/man2/time.2.html)). Which is quite diferent than the time `SYSTEMTIME` structure stores, that is since [1601-01-01 00:00:00](https://docs.microsoft.com/en-us/windows/win32/api/minwinbase/ns-minwinbase-systemtime), maybe we could do something like [this](https://stackoverflow.com/a/26085827)?

Reply:
It would be something like what we have in: [https://github.com/expertisesolutions/efl/blob/devs/joaoantoniocardoso/native-rebased/ecore/src/lib/evil/evil_time.h#L28](url)

Reply:
This link isn't working.

Reply:
@Coquinho the link is wrong but the written url is right, copy and past it to your navbar.

Reply:
lol, thanks

Reply:
@ricardocvel, yes, I think we could use that implementation.

Reply:
@joaoantoniocardoso , @Coquinho I still haven't been able to test whether the files that reference gettimeofday () are really getting the evil_time.c function

Reply:
you can do the following: 
- `git checkout devs/joaoantoniocardoso/native-rebased/ecore`
- `git revert 984c353`
- try to build, (it should fail because of gettimeofday is missing), then 
- `git pull devs/expertise-ricardo/native-rebased/ecore-gettimeofday` and
- try to build again and see if it worked